hathor/indexes/partial_rocksdb_tips_index.py

```diff
@@ -127,9 +139,18 @@ def _from_key(self, key: bytes) -> Interval:
         assert len(tx_id) == 32
         return Interval(_from_db_value(begin), _from_db_value(end), tx_id)
 
-    def init_start(self) -> None:
+    def init_start(self, indexes_manager: 'IndexesManager') -> None:
         log = self.log.new(index=f'tips-{self._name}')
-        total = int(self._db.get_property(b'rocksdb.estimate-num-keys', self._cf))
```

Comment:
Why can't we use the rocksdb estimate key?

Reply:
We certainly _can_, but it can be frequently way off, like I've seen it go to 130% completion because the estimate was off.

Reply:
I just discussed about it with @jansegre. We will refactor it later on a separate PR because we will need to change the IndexManager to let the index decide what type of vertices it will allow.